local docker test environment

krisss85 opened the pull request:
The purpose of this PR is to setup local docker test environment

bolkedebruin commented on docker/docker-compose.yml:
```diff
@@ -0,0 +1,50 @@
+
+services:
+  mysql:
+      image: mysql:5.7
```
Are you doing this for persistence? I can run keycloak (quay.io/keycloak/keycloak , which is the recommended one: https://www.keycloak.org/getting-started/getting-started-docker) without it

krisss85 replied:
But then you face a problem that you have to restart the container
`keycloak_1  | Added 'admin' to '/opt/jboss/keycloak/standalone/configuration/keycloak-add-user.json', restart server to load user
keycloak_1  | -b 0.0.0.0`

bolkedebruin replied:
Check, its just nice to have it a bit lighter.

krisss85 replied:
sure thing, it should be addressed now, i created a separate realm for rdpgw

bolkedebruin commented on docker/docker-compose.yml:
```diff
@@ -0,0 +1,50 @@
+        DB_USER: keycloak
+        DB_PASSWORD: password
+        KEYCLOAK_USER: admin
+        KEYCLOAK_PASSWORD: Pa55w0rd
```
I suggest just using admin, as this is for development and no one will remember this one after a while ;-)

krisss85 replied:
:+1:

bolkedebruin commented on docker/docker-compose.yml:
```diff
@@ -0,0 +1,50 @@
+        test: ["CMD", "curl", "-f", "http://keycloak:8080"]
+        interval: 30s
+        timeout: 10s
+        retries: 10
```
add a new line plz

krisss85 replied:
all good now?

bolkedebruin commented on docker/rdpgw.yaml:
```diff
@@ -0,0 +1,30 @@
+ DisablePrinter: true
+ DisablePort: true
+ DisablePnp: true
+ DisableDrive: true
```
new line please

krisss85 replied:
all good now?

krisss85 commented:
@bolkedebruin i have addressed all the comments, it is now fully fledged self-contained testing env, please try it out according to readme and let me know if there is anything missing still

bolkedebruin commented:
Awesome looks good!
